src/runtimes/node/bundlers/zisi/src_files.ts

```diff
@@ -87,6 +88,19 @@ const getDependencies = async function ({
   }
 }
 
+const paperwork = async (path: string) => {
+  const modules = await precinct.paperwork(path, { includeCore: true })
```

Comment:
If I understand correctly, by using `includeCore: true` instead of `includeCore: false`, we're taking over the job of filtering the core modules, which you're doing with `isBuiltinModule`. I see the precinct seems to use [a different detection mechanism](https://github.com/dependents/node-precinct/blob/253d4b05c1d26ac662b76390046bc48d4814c5b2/index.js#L21), though.

Is there a risk this will cause regressions, with some modules suddenly being evaluated differently?

Reply:
that's a good point - we should stick to process.binding if possible. isBuiltinModule is using a [similar mechanism](https://github.com/sindresorhus/builtin-modules/blob/main/index.js) under the hood, but there could be subtle differences ...

Reply:
done in https://github.com/netlify/zip-it-and-ship-it/pull/1069/commits/c75179fea6142e65f0facd3174625829d5a66408